discovery-infra/test_infra/consts.py

```diff
@@ -21,6 +21,7 @@
 WAIT_FOR_BM_API = 900
 NAMESPACE_POOL_SIZE = 15
 PODMAN_FLAGS = "--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file"
+LOG_FOLDER = "/tmp/assisted_test_infra_logs"
```

Comment:
Don't we want to pass this parameter from pytest? then it would be easy to get archived

Reply:
What do you mean? As a command line argument?

Reply:
question - its a skipper folder? or local?

Reply:
/tmp is mounted to the container
https://github.com/openshift/assisted-test-infra/blob/master/skipper.yaml#L11

Reply:
> What do you mean? As a command line argument?

Yes,
So if you would want to run on another environment or something, you will be able to change the path of the log.

Reply:
changed